Mail/Views/Thread List/ThreadListHeader.swift

```diff
@@ -99,9 +102,17 @@ struct ThreadListHeader: View {
                 if !networkMonitor.isConnected {
                     NoNetworkView()
                 }
-                if let lastUpdateText = folderObserver.lastUpdateText {
-                    Text(MailResourcesStrings.Localizable.threadListHeaderLastUpdate(lastUpdateText))
-                        .textStyle(.bodySmallSecondary)
+                if isRefreshing {
+                    HStack {
+                        ProgressView().padding(.trailing, 2)
```

Comment:
```suggestion
                    HStack(spacing: 2) {
                        ProgressView()
```
You can set the spacing between the two views in the HStack directly ;)